src/main/java/net/minecraftforge/event/world/ChunkWatchEvent.java

```diff
@@ -57,6 +71,16 @@ public EntityPlayerMP getPlayer()
         return player;
     }
 
+    /**
+     * The world of the chunk.
+     * @return
```

Comment:
This javadoc should probably be `@return The world of the chunk.`
Right now it looks unfinished.